frontend/src/metabase-lib/expression.ts

```diff
@@ -24,3 +32,23 @@ export function expressionableColumns(
 ): ColumnMetadata[] {
   return ML.expressionable_columns(query, stageIndex, expressionPosition);
 }
+
+export function expressionParts(
+  query: Query,
+  stageIndex: number,
+  clause: ExpressionClause | FilterClause,
+) {
+  return ML.expression_parts(query, stageIndex, clause);
+}
+
+export function expressionClause(
+  operator: ExpressionOperator,
+  options: ExpressionOptions | null,
+  args: (ExpressionArg | ExpressionClause)[],
+): ExpressionClause {
+  return ML.expression_clause(operator, options, args);
+}
+
+export function isColumnMetadata(arg: unknown): arg is ColumnMetadata {
+  return ML.is_column_metadata(arg);
+}
```

Comment:
Not sure if `expressions` is the right file for it, maybe `metabase-lib/metadata` would be better?